common/libs/nodes/rpc/index.ts

```diff
@@ -5,6 +5,22 @@ import { Wei } from 'libs/units';
 import { INode, TxObj } from '../INode';
 import RPCClient from './client';
 import RPCRequests from './requests';
+import { isValidGetBalance } from '../../validators';
+
+// TODO - understand response values
+// "RPC" requests will sometimes resolve with 200, but contain a payload informing there was an error.
+function ensureOkResponse(response: any) {
```

Comment:
May be redundant since validation should catch an error response (it *should* have a different shape as the success response.